dotcom-rendering/src/web/experiments/tests/integrate-smart-test.ts

```diff
@@ -0,0 +1,20 @@
+import type { ABTest } from '@guardian/ab-core';
+
+export const integrateSmart: ABTest = {
+	id: 'IntegrateSmart',
+	author: 'Fred OBrien (@frederickobrien)',
```

Comment:
Just to bring this in line with Frontend...
```suggestion
	author: 'Commercial Dev (@guardian/commercial-dev)',
```

Reply:
Good call. Thanks!